scripts/sql/apolloconfigdb.sql

```diff
@@ -345,7 +345,8 @@ CREATE TABLE `ReleaseHistory` (
   PRIMARY KEY (`Id`),
   KEY `IX_Namespace` (`AppId`,`ClusterName`,`NamespaceName`,`BranchName`),
   KEY `IX_ReleaseId` (`ReleaseId`),
-  KEY `IX_DataChange_LastTime` (`DataChange_LastTime`)
+  KEY `IX_DataChange_LastTime` (`DataChange_LastTime`),
```

Comment:
Please add a folder `v200-v210` in [delta](https://github.com/apolloconfig/apollo/tree/master/scripts/sql/delta) and add a upgrade script there.

Reply:
Done.

Reply:
> Please add a folder `v200-v210` in [delta](https://github.com/apolloconfig/apollo/tree/master/scripts/sql/delta) and add a upgrade script there.

Done.